apps/web/pages/index.vue

```diff
@@ -67,6 +69,26 @@ const { data, fetchPageTemplate } = useHomepage();
 const { fetchCategoryTemplate } = useCategoryTemplate();
 const { showNewsletter } = useNewsletter();
 
+const addNewBlock = (index: number, position: number) => {
+  const insertIndex = position === -1 ? index : index + 1;
+  const updatedBlocks = [...data.value.blocks];
+
+  updatedBlocks.splice(insertIndex, 0, {
+    name: 'UiMediaCard',
+    options: {
```

Comment:
**issue**: Please check the language and insert the corresponding options.

Reply:
I've added that in latest commit